app/jinja_filters.py

```diff
@@ -25,8 +25,15 @@ def format_number(value):
     return ''
 
 
+@evalcontextfunction
 @blueprint.app_template_filter()
-def format_currency(value, currency='GBP'):
+def format_currency(context, value, currency='GBP'):
+    currency_value = get_formatted_currency(value, currency)
+    result = "<span class='date'>{currency_value}</span>".format(currency_value=currency_value)
```

Comment:
This will use the `date` class in the span for now. A new card exists to use a new utility class that has been added to the pattern library. https://github.com/ONSdigital/sdc-global-design-patterns/pull/161